net/netfilter/nf_tables_api.c

```diff
@@ -8391,18 +8413,25 @@ static void nf_tables_cleanup(struct net *net)
 static int nf_tables_abort(struct net *net, struct sk_buff *skb,
 			   enum nfnl_abort_action action)
 {
-	int ret = __nf_tables_abort(net, action);
+	struct nftables_pernet *nft_net = nft_pernet(net);
```

Comment:
Hmm... I think this will be resolved in my next push to branch, right now the code there is identical to the 534 build.

I'll leave this unresolved for now and we can revisit after the next push.

Reply:
You're correct I made a goof on the old code represented and is likely correct because we probably don't want to unlock the parent `nft_net`